pkgs/applications/video/vlc/default.nix

```diff
@@ -196,6 +196,11 @@ stdenv.mkDerivation rec {
       url = "https://code.videolan.org/videolan/vlc/uploads/eb1c313d2d499b8a777314f789794f9d/0001-Add-lssl-and-lcrypto-to-liblive555_plugin_la_LIBADD.patch";
       sha256 = "0kyi8q2zn2ww148ngbia9c7qjgdrijf4jlvxyxgrj29cb5iy1kda";
     })
+    # patch to build with recent libplacebo
+    (fetchpatch {
+      url = "https://code.videolan.org/videolan/vlc/-/merge_requests/3027.patch";
+      hash = "sha256-aV+YT1l0ND/USoIIpxcPhdIlP/06J2FxVW4uArS8j88=";
```

Comment:
```console
error: hash mismatch in fixed-output derivation '/nix/store/3b1xm0v22yizbr8nkd2p2hnyqg956ngc-3027.patch.drv':
         specified: sha256-aV+YT1l0ND/USoIIpxcPhdIlP/06J2FxVW4uArS8j88=
            got:    sha256-Zz+g75V6X9OZI3sn614K9Uenxl3WtRHKSdLkWP3b17w=
error: 1 dependencies of derivation '/nix/store/mp38cfqhwj1m02a118vhrmh5z5hkfkgk-libvlc-3.0.18.drv' failed to build
```

Did GitLab change the hash for a merged MR, or did we miscalculate the hash somewhere down the fetchpatch hill?

Reply:
Already fixed in another pr.